test/integration/account_settings_test.go

```diff
@@ -0,0 +1,69 @@
+package integration
+
+import (
+    "context"
+    "testing"
+
+    "github.com/linode/linodego"
+)
+
+func TestAccountSettings_Get(t *testing.T) {
+    client, teardown := createTestClient(t, "fixtures/TestAccountSettings")
+    defer teardown()
+
+    settings, err := client.GetAccountSettings(context.Background())
+    if err != nil {
+        t.Fatalf("Error getting Account Settings, expected struct, got error %v", err)
+    }
+
+    if settings.BackupsEnabled != true {
```

Comment:
optional: We now import the `testify/require` package so you can simplify these assertions to something like `require.False(t, settings.BackupEnabled)`. 

This is mostly a stylistic suggestion so feel free to ignore 🙂